templates/repo/issue/view_content/comments.tmpl

```diff
@@ -91,7 +104,59 @@
 
 			<div class="detail">
 				<span class="octicon octicon-git-commit"></span>
-				<span class="text grey">{{.Content | Str2html}}</span>
+				<span class="text grey"><a href="{{.RefURL}}"><span class="title has-emoji">{{.RefCommitMessage | Escape}}</span> ({{.RefCommitShortSHA | Escape}})</a></span>
+			</div>
+		</div>
+	{{else if and (eq .Type 5) .RefExists}}
+		<div class="event">
+			<span class="octicon octicon-bookmark"></span>
+			<a class="ui avatar image" href="{{.Poster.HomeLink}}">
+				<img src="{{.Poster.RelAvatarLink}}">
+			</a>
+			<span class="text grey"><a href="{{.Poster.HomeLink}}">{{.Poster.Name}}</a> {{$.i18n.Tr "repo.issues.comment_ref_at" .EventTag $createdStr | Safe}}</span>
+
+			<div class="detail">
+				<span class="octicon octicon-comment-discussion"></span>
+				<span class="text grey"><a href="{{.RefURL}}"><span class="title has-emoji">{{.RefIssue.Title | Escape}}</span> (#{{.RefIssue.Index}})</a></span>
+			</div>
+		</div>
+	{{else if and (eq .Type 6) .RefExists}}
+		<div class="event">
+			<span class="octicon octicon-bookmark"></span>
+			<a class="ui avatar image" href="{{.Poster.HomeLink}}">
+				<img src="{{.Poster.RelAvatarLink}}">
+			</a>
+			<span class="text grey"><a href="{{.Poster.HomeLink}}">{{.Poster.Name}}</a> {{$.i18n.Tr "repo.issues.pull_ref_at" .EventTag $createdStr | Safe}}</span>
+
+			<div class="detail">
+				<span class="octicon octicon-git-pull-request"></span>
+				<span class="text grey"><a href="{{.RefURL}}"><span class="title has-emoji">{{.RefIssue.Title | Escape}}</span> (#{{.RefIssue.Index}})</a></span>
+			</div>
+		</div>
+	{{else if and (eq .Type 23) .RefExists}}
+		<div class="event">
+			<span class="octicon octicon-bookmark"></span>
+			<a class="ui avatar image" href="{{.Poster.HomeLink}}">
+				<img src="{{.Poster.RelAvatarLink}}">
+			</a>
+			<span class="text grey"><a href="{{.Poster.HomeLink}}">{{.Poster.Name}}</a> {{$.i18n.Tr "repo.issues.review_ref_at" .EventTag $createdStr | Safe}}</span>
+
+			<div class="detail">
+				<span class="octicon octicon-comment-discussion"></span>
+				<span class="text grey"><a href="{{.RefURL}}"><span class="title has-emoji">{{.RefIssue.Title | Escape}}</span> (#{{.RefIssue.Index}})</a></span>
+			</div>
+		</div>
+	{{else if and (eq .Type 24) .RefExists}}
```

Comment:
I know this file previously used hardcoded ints to do an equality check against type, but could you change it to do an equality check where `24` could easily be determined? (same applies for other checks too)